# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../poppler/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
Include a missing header - wonder how this compiles on the devs machine...
	--jsaw

--- poppler-0.3.1/poppler/OutputDev.h.orig	2005-05-12 17:17:48.000000000 +0200
+++ poppler-0.3.1/poppler/OutputDev.h	2005-05-12 17:16:40.000000000 +0200
@@ -16,6 +16,7 @@
 #include <poppler-config.h>
 #include "goo/gtypes.h"
 #include "CharTypes.h"
+#include "Object.h"
 
 class GooString;
 class Object;
